client/src/componenets/App.jsx

```diff
@@ -12,13 +12,16 @@ import axios from 'axios';
 // dispatch({ type: 'MAIN_IMAGE', image: res.data[0].game_photo })
 
 const App = () => {
-  const [currentGameId] = useState(1);
+  const [currentGameId] = useState(2);
```

Comment:
I'm not familiar with some of the syntax here such as "useState" and "useEffect" but it seems to be for React hooks? I like that you've taken the time to learn that and are trying to implement that here.

After looking at the main App code file I do have 2 suggestions: renaming your main component from "App" to something like "GameCarousel" or something that will make it distinct when we assemble all of our components onto the proxy (we mentioned this tonight so you're probably planning to do that already) and separating out your CoverPhoto component to its own file even though its barely any lines of code. I think its just a best practice for components to be in their own files and will be cleaner for jest/enzyme testing as well since its recommended that for each component you test, you create a new file and have that file live in the same directory as that component. Besides that, things look really good here!